iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -132,13 +130,15 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationMessageCellLay
     /// Returns the frame that a message drop zone should have, based on this cell's contents.
     /// The frame is in the coordinate space of the passed in view.
     func getMessageDropZoneFrame(convertedTo targetView: UIView) -> CGRect {
-        if let frontCellIndex = self.collectionLayout
-            .getFrontmostItemIndexPath(inSection: ConversationMessageSection.currentUserMessages.rawValue),
-           let frontUserCell = self.collectionView.cellForItem(at: frontCellIndex) {
-
-            let overlayRect = frontUserCell.convert(frontUserCell.bounds, to: self)
-            return self.convert(overlayRect, to: targetView)
-        }
+        // TODO: Restore this
+        return .zero
+//        if let frontCellIndex = self.collectionLayout
+//            .getFrontmostItemIndexPath(inSection: ConversationMessageSection.currentUserMessages.rawValue),
+//           let frontUserCell = self.collectionView.cellForItem(at: frontCellIndex) {
+//
+//            let overlayRect = frontUserCell.convert(frontUserCell.bounds, to: self)
+//            return self.convert(overlayRect, to: targetView)
+//        }
```

Comment:
I need to update the code that determines the drop zone frame for the new layout.